usecases/Concrete/workflow/joerg_example/wf_run_demonstrator.cwl

```diff
@@ -0,0 +1,41 @@
+class: Workflow
```

Comment:
I was no able to visualize the workflow, is that correctly working?

Reply:
probably not